src/js/components/AppListComponent.jsx

```diff
@@ -80,12 +113,42 @@ var AppListComponent = React.createClass({
     });
   },
 
+  getAppsInGroup: function () {
+    var apps = this.state.apps;
+    var currentGroup = this.props.currentGroup;
+
+    return lazy(apps)
+      .filter((app) => app.id.startsWith(currentGroup))
+      .reduce((memo, app) => {
```

Comment:
Addressed in https://github.com/mesosphere/marathon-ui/commit/3bdb4e315838c880b36ba717fb13cee5c8ec0bf1